test/tests/device/isiPadOs.js

```diff
@@ -0,0 +1,77 @@
+/* @flow */
+
+import { isIpadOs } from "../../../src/device";
+
+describe("isIpadOs", () => {
+  beforeEach(() => {
+    window.navigator = {};
+  });
+
+  it("should return true when userAgent is Safari and has multiple maxTouchPoints", () => {
+    window.navigator.userAgent =
+      "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.2 Safari/605.1.15";
+    // eslint-disable-next-line compat/compat
+    window.navigator.maxTouchPoints = 5;
+    const bool = isIpadOs();
+    if (!bool) {
+      throw new Error(`Expected true, got ${JSON.stringify(bool)}`);
+    }
+  });
+
+  it("should return true when userAgent is Firefox and has multiple maxTouchPoints", () => {
+    window.navigator.userAgent =
+      "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Safari/605.1.15";
+    // eslint-disable-next-line compat/compat
+    window.navigator.maxTouchPoints = 5;
+    const bool = isIpadOs();
+    if (!bool) {
```

Comment:
```suggestion
    if (bool !== true) {
```
I'd suggest explicitly equating to a boolean for these